renderer/components/Onboarding/Steps/ConnectionConfirm.js

```diff
@@ -46,7 +46,7 @@ class ConnectionConfirm extends React.Component {
       connectionString,
       startLnd,
     } = this.props
-
+    connectionString = connectionString && connectionString.trim()
```

Comment:
Instead of doing the trim here at point of usage, perhaps we could do it at the source in `LndConnectionStringInput` using `Informed::mask`?

Reply:
This sounds like a better approach. Will work on it